tests/test_wps_QDM.py

```diff
@@ -29,6 +28,5 @@ def test_wps_QDM(obs_file, gcm_file, var, end_date, num_cores):
             f"varname={var};"
             f"out_file={out_file.name};"
             f"num_cores={num_cores};"
-            f"end_date={end_date};"
```

Comment:
So, I guess this [comment](https://github.com/pacificclimate/ClimDown/blob/master/R/QDM.R#L190) in ClimDown QDM is wrong?

Reply:
I believe so, but I can check with James

Reply:
Actually, taking a look at the code, it is used in qdm, I will move it